src/main/ipc/MessageHandler.js

```diff
@@ -68,30 +72,6 @@ class MessageHandler {
   getScreenSize() {
     return screen.getPrimaryDisplay().workAreaSize;
   }

```

Comment:
Maybe create an APP channel for these?

Reply:
Leave these as general IPCs atm, if more get added consider adding them as a separate channel.